apcd-cms/src/apps/exception/templates/exception_submission_form/exception_threshold_form.html

```diff
@@ -294,6 +311,8 @@ <h4>Acknowledgment of Terms</h4>
                     <div id="help-text-threshold-requested_${exceptions}" class="help-text">
                     </div>
                   </div>
+                  </div>
+                  </div>
                   </div>`
```

Comment:
Are all of these needed? Not sure if I'm missing the opening `<div>` tags for these.

Reply:
Good catch - removing the last two div